crates/symbolicator-service/src/services/download/s3.rs

```diff
@@ -155,93 +163,78 @@ impl S3Downloader {
         tracing::debug!("Fetching from s3: {} (from {})", &key, &bucket);
 
         let source_key = &file_source.source.source_key;
-        let client = self.get_s3_client(source_key);
-        let request = client.get_object(rusoto_s3::GetObjectRequest {
-            key: key.clone(),
-            bucket: bucket.clone(),
-            ..Default::default()
-        });
-
-        let source = RemoteDif::from(file_source);
+        let client = self.get_s3_client(source_key).await;
+        let request = client
+            .get_object()
+            .bucket(bucket.clone())
+            .key(key.clone())
+            .send();
+
+        let source = RemoteDif::from(file_source.clone());
```

Comment:
I believe if you clean up all the `println!`s below, you can avoid these `clone`s as well.

Reply:
Fixed by converting to clones to references.